src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1997,6 +1999,7 @@
 
   <Node Name="LocalDeclarationStatementSyntax" Base="StatementSyntax">
     <Kind Name="LocalDeclarationStatement"/>
+    <Field Name="AttributeLists" Type="SyntaxList&lt;AttributeListSyntax&gt;" Override="true"/>
```

Comment:
Do we have a requirement that derived syntax nodes "override" their base fields in here?

Reply:
Yes.  It's how the generator works.  the member in the abstract base class just signifies that hte member exists, but the subclasses must override because the base class has no idea *where* in the field list the member should go.  This order is inherantly part of the definition of a node that is critical for everything about the layout of it, what is signifies, how it ToStrings, etc. etc.